charts/graph-node/out.yaml

```
@@ -0,0 +1,1473 @@
+            - name: "STORE_CONNECTION_POOL_SIZE"
+              value: "10"
+            - name: "PINAX_JWT"
+              valueFrom:
+                secretKeyRef:
+                  name: "pinax-jwt-secret"
+                  key: "jwt"
+                  optional: false
+            - name: "PRIMARY_SUBGRAPH_DATA_PGPASSWORD"
+              valueFrom:
+                secretKeyRef:
+                  name: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)